CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.6.2 / 2017-05-11
+
+* [BUGFIX] Fix potential memory leak in Kubernetes service discovery
+
+## 1.5.3 / 2017-05-11
```

Comment:
I'm used to read changelogs sorted by version, sorting by date looks confusing. Once a release is in the past, the exact date is really not that important to me.